week-3/extended-project/nowhere/serve.rb

```diff
@@ -0,0 +1,58 @@
+require 'socket'
+require "erb"
+require_relative 'routes.rb'
+require_relative 'members.rb'
+
+port = 3000
+server = TCPServer.new port
+
+puts "Running server 🚀  in http://localhost:#{port}"
+
+loop do
+  session = server.accept
+  request = []
+  while (line = session.gets) && (line.chomp.length > 0)
+    request << line.chomp
+  end
+
+  puts request[0]
+
+  http_method, path, protocol = request[0].split(' ')
+
+  puts path
+
+  content = 'text/html'
+  status = "200 OK"
+
+  if PAGES.keys.include? path
+    template = File.read('views/' + PAGES[path])
+  elsif path.include? 'member'
+    member = path.split("=")[1]
+    if TEAM.key?(member.to_sym)
+      template = File.read('views/member.erb')
+    else
+      status = "404 Not Found"
+      template = File.read('views/' + PAGE_NOT_FOUND)
+    end
+  elsif path.match(/.css/)
+    template = File.read('public' + path)
```

Comment:
You could consider to validate before reading the value sended by url since if I add the text "/style.css" to your url, the server crushes